security/remote_devices/docker-swarm/gettoken.sh

```diff
@@ -0,0 +1 @@
+docker exec -ti `docker ps  --filter name=consul | awk '{ print $1 }' | tail -n 1` cat /tmp/edgex/secrets/device-virtual/secrets-token.json
```

Comment:
Why is this needed when the other script has to be run first and could just echo it?

Reply:
`gettoken.sh` is run on machine A and `puttoken.sh` is run on machine B to copy it.